### DISCUSSION TITLE
Lawsuit financing as "managing risk"

### DISCUSSION THREAD

**Discussion by bretweinraub** [risk-first/website#77] [Team Posts]
"Whether you're hired by the plaintiff or by the defendant, what you're really doing is managing risk."

https://www.npr.org/templates/transcript/transcript.php?storyId=766556249&t=1570824440157

Here you go @robmoffat, my first contribution, which was riding my mtn bike listening to this podcast.

**Comment by robmoffat** [comment]
Sounds like gripping listening, Bret!
